models/delta.coffee

```diff
@@ -90,15 +92,19 @@ class DeltaStream extends EventEmitter
   open: () ->
     @close()
     path = "/delta/streaming"
+    excludeTypes = @params.excludeTypes ? []
+    includeTypes = @params.includeTypes ? []
+
     queryObj =
       cursor: @cursor
-    queryObj.exclude_types = @excludeTypes.join(',') if @excludeTypes?.length > 0
-    queryStr = querystring.stringify(queryObj)
-    path += '?' + queryStr
+    queryObj.exclude_types = excludeTypes.join(',') if excludeTypes.length > 0
```

Comment:
This looks great—my only feedback is that we could make this a bit more concise, since we `? []` and then say `.length > 0`.

Maybe:
```
queryObj.exclude_types = (@params.excludeTypes ? []).join(',')
```

The only thing is that exclude would be '' if the array was zero length.. hmm.